.travis.yml

```diff
@@ -1,10 +1,21 @@
+# Wire up travis
 language: python
-python:
-    - 2.7
+
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=py27
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
+    - python: 3.8
+      env: TOXENV=py38
```

Comment:
I would suggest [tox-travis](https://pypi.org/project/tox-travis/) to avoid repetition.

(The main reason we don't normally use tox-travis in other zopefoundation repos is because we want to measure test coverage and upload it to coveralls in our .travis.yml, but not in tox.ini, but AFAICT this is not currently set up for z3c.sqlalchemy.)